SDWebImage/Core/SDDiskCache.m

```diff
@@ -308,17 +308,17 @@ - (void)moveCacheDirectoryFromPath:(nonnull NSString *)srcPath toPath:(nonnull N
     if (str == NULL) {
         str = "";
     }
-    unsigned char r[CC_MD5_DIGEST_LENGTH];
-    CC_MD5(str, (CC_LONG)strlen(str), r);
+    unsigned char r[CC_SHA256_DIGEST_LENGTH];
+    CC_SHA1(str, (CC_LONG)strlen(str), r);
```

Comment:
Why `CC_SHA1` here instead of `CC_SHA256`? Looks like a typo.

Reply:
This pr maybe discard.